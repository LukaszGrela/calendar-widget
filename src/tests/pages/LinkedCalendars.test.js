import React from 'react';
import {shallow} from 'enzyme';
import LinkedCalendars from '../../pages/LinkedCalendars';

test('should render correctly', () => {
    const wrapper = shallow(<LinkedCalendars />);

    expect(wrapper).toMatchSnapshot();
})