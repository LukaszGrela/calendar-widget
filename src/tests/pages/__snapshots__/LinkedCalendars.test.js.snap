// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<section
  className="linked-calendars"
>
  <article>
    <p>
      React Calendar Widget example.
    </p>
  </article>
  <article
    className="widgets"
  >
    <CalendarWidget
      className="linked currentMonth"
      currentMonth={"1970-01-01T00:00:00.000Z"}
      onDateChanged={[Function]}
      todayDate={"1970-01-01T00:00:00.000Z"}
    />
    <CalendarWidget
      className="linked nextMonth"
      currentMonth={"1970-02-01T00:00:00.000Z"}
      onDateChanged={[Function]}
      todayDate={"1970-01-01T00:00:00.000Z"}
    />
  </article>
  <nav>
    <NavLink
      activeClassName="active"
      ariaCurrent="true"
      to="/"
    >
      Home
    </NavLink>
  </nav>
</section>
`;
